src/main/scala/io/github/spark_redshift_community/spark/redshift/RedshiftJDBCWrapper.scala

```diff
@@ -225,9 +224,9 @@ private[redshift] class JDBCWrapper {
       throw new IllegalArgumentException(s"Did not find registered driver with class $driverClass")
     }
     val properties = new Properties()
-    credentials.foreach { case(user, password) =>
-      properties.setProperty("user", user)
-      properties.setProperty("password", password)
+    mergedParameters.parameters.foreach { case (key, value) =>
+      log.info(s"key: ${key} value ${value}")
+      properties.setProperty(key, value)
```

Comment:
Here you will log every key value, not just the `user`, `password` ones like in the original implementation 🤔 Do you need to be logging them for "Prod" or is this a testing remnant ? I am not sure we should be logging them cause they may contain creds 🤷‍♂️

Reply:
fuuuuuuck will remove, this is for my testing 🤦, good catch!